python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -76,17 +76,21 @@ def to_file(j, fobj=None, ignore_subs=''):
 # * AssertionError (corruption: multiple objects in <root>...</root>
 # * Exception (probably corrupted data problem)
 
-def _raw_from_file(f):
+def _raw_from_file(f, obj):
     # logger.debug('----------------------------')
     ###logger.debug('Parsing file: %s',f.name)
     xml_content = f.read()
-    obj, errors = Loader().parse(xml_content)
+    xml_lines = xml_content.splitlines()
+    for line in xml_lines:
+        line = line.lstrip()
+    xml_content = '\n'.join(xml_lines)
+    obj, errors = Loader(obj).parse(xml_content)
```

Comment:
I feel that this change causes a lot more complexity for very little gain. Maintainability always wins out over minor speed increases (and in fact most other things) in the long term. If we're loading a file with 1000 `GangaObject`s in it, this perhaps slightly speeds up the loading of just one of them. It's probably also cleaner to know we're always creating a `Job` object (or whatever) from scratch to avoid loitering cached data.